system/services/email/EmailStatsService.cfc

```diff
@@ -415,10 +415,14 @@ component {
 		var turnedOnDate = Now();
 		var dateFilter   = { filter="email_template_send_log_activity.datecreated <= :datecreated", filterParams={ datecreated=turnedOnDate } };
 
-		templateDao.updateData( id=arguments.templateId, data={
-			  stats_collection_enabled    = true
-			, stats_collection_enabled_on = turnedOnDate
-		} );
+		templateDao.updateData(
+			  id      = arguments.templateId
+			, isDraft = $getPresideObjectService().objectRecordHasDraft( objectName="email_template", recordId=arguments.templateId )
```

Comment:
This isn't quite right. `hasDraft` is whether or not the template has any drafts - not that it IS a draft. I have made a separate commit and PR for this - closing this one.